packages/beacon-node/test/unit/chain/validation/lightClientFinalityUpdate.test.ts

```diff
@@ -80,9 +84,11 @@ describe("Light Client Finality Update validation", function () {
       return ssz.altair.LightClientFinalityUpdate.defaultValue();
     };
 
-    await expectRejectedWithLodestarError(
-      Promise.resolve(validateLightClientFinalityUpdate(config, chain, lightClientFinalityUpdate)),
```

Comment:
Can you try this syntax to see if it works with `expectRejectedWithLodestarError`?
```ts
await expectRejectedWithLodestarError(
  (async () => validateLightClientFinalityUpdate(config, chain, lightClientFinalityUpdate))(),
  LightClientErrorCode.FINALITY_UPDATE_NOT_MATCHING_LOCAL
)
```

Reply:
The function being tested is not `asynchronous`. `expectRejectedWithLodestarError` expects an asynchronous input. Your suggestion feels like shoe-horning the test to satisfy  `expectRejectedWithLodestarError`'s need. Is there any benefit to doing that?